src/code/file_menus.asm

```diff
@@ -146,60 +146,66 @@ func_001_4852::
     pop  de                                       ; $4891: $D1
     ld   a, $05                                   ; $4892: $3E $05
 
-jr_001_4894::
+.drawEmptyRowLoop
+    ; Draw the empty row above the save slot name;
+    ; might contain diacritics
     ldh  [hMultiPurpose0], a                      ; $4894: $E0 $D7
     ld   a, [de]                                  ; $4896: $1A
     and  a                                        ; $4897: $A7
 
 IF LANG_EN
-    jr   .jr_489D                                 ; $4898: $18 $03
+    jr   .selectEmptyRowTile                      ; $4898: $18 $03
+    ; Unreachable code:
     dec  a                                        ; $489A: $3D
     and  $C0                                      ; $489B: $E6 $C0
 ELSE
-    jr   z, .jr_489D
+    jr   z, .selectEmptyRowTile
     dec  a
     push hl
     push bc
     ld   c, a
     ld   b, $00
-    call ReadDialogBankFromTable
-IF __PATCH_9__
+    call ReadTileValueFromDiacriticsTable
+IF __DO_CHECK_DIACRITICS__
```

Comment:
Maybe it's time to also rename the patch to something with a wider meaning – like `__FEAT_DIACRITICS__` or `__DIACRITICS_SUPPORT__` ?